### DISCUSSION TITLE
How to show a checkerboard background in transparent images.

### DISCUSSION THREAD

**Discussion by nephewtom** [d2phap/ImageGlass#1295] [General] [answered]
Hello!
I would like to know how to show a checkerboard background in transparent images.
Have read this feature request issue: [Show checked background on transparent images only](https://github.com/d2phap/ImageGlass/issues/259).

But I am not able to see it with this PNG image:
![redis-original](https://user-images.githubusercontent.com/2485147/155360607-c9bebb61-f710-441f-9300-66ac0fa42573.png)

This is ImageGlass:
![image](https://user-images.githubusercontent.com/2485147/155360967-a6d659d3-4d8a-416f-8356-a9b565f449a2.png)

With this option set:
![image](https://user-images.githubusercontent.com/2485147/155361302-80c4ec7c-2580-481a-8326-a9a178a8c02c.png)

And I would like to get something like this (opening with paint.net):
![image](https://user-images.githubusercontent.com/2485147/155361487-0f5be747-fd2d-4d13-a93c-e150b2238f41.png)

Is it possible to achieve that?

**Comment by d2phap** [comment, marked as answer]
You can click on this button:
![image](https://user-images.githubusercontent.com/3154213/155371878-099f335c-fab1-4da1-b02f-d1f20a36e958.png)

Or press <kbd>B</kbd>

Or open main menu > Layout > Show checkerboard

![image](https://user-images.githubusercontent.com/3154213/155371974-f3e8e3af-7ad7-470b-8c38-c704d1dee200.png)

**Reply by nephewtom** [reply]
Great! 🥇 
That made it.
Thank you very much!
🙏

**Comment by fire-eggs** [comment]
Here's IG 8.5.1.22 showing said image (light grey theme):

![image](https://user-images.githubusercontent.com/9809727/155371535-62d02b8e-fb23-4c57-aebd-f6bfdfca8232.png)

And here it is with Kobe theme:

![image](https://user-images.githubusercontent.com/9809727/155371965-a3c0fcb4-0740-4c80-ac2a-54d36a8ecbb3.png)
